### PR TITLE
Add multi server support

### PR DESCRIPTION
**Pull request by 577fkj**

### CODE REVIEW

**Comment by mingjun97** [comment]
Amazing! Thanks for your contribution!

**Comment by mingjun97** [comment]
Hi, I just update the mojoconsole to adapt breaking changes from Grasscutter. Could you merge the latest mojoconsole into your repo and test it? Thank you so much!

**Review by mingjun97 — APPROVED** [review]

File: gc-plugin/src/main/java/com/mojo/consoleplus/ConsolePlus.java [hunk 1]
```diff
@@ -12,20 +14,33 @@
...
+import emu.grasscutter.server.event.HandlerPriority;
+import emu.grasscutter.server.event.player.PlayerJoinEvent;
+import emu.grasscutter.server.event.player.PlayerQuitEvent;
 import io.javalin.http.staticfiles.Location;
 import emu.grasscutter.plugin.PluginConfig;
 import static emu.grasscutter.Configuration.PLUGIN;
 import static emu.grasscutter.Configuration.HTTP_POLICIES;
 
 import com.mojo.consoleplus.config.MojoConfig;
+import org.slf4j.Logger;
```
[the remark this answers is not in the document]
Comment (577fkj) [reply gc-plugin/src/main/java/com/mojo/consoleplus/ConsolePlus.java:27]: There is some confusion in the calling of some codes. I'll optimize it when I have time

[the remark this answers is not in the document]
Comment (mingjun97) [reply gc-plugin/src/main/java/com/mojo/consoleplus/ConsolePlus.java:27]: Great! I've review all the codes. Overall it's good. Let me know if you think it's ready to merge.

**Comment by 577fkj** [comment]
Now it can be merged

**Pull request merged**